ros/api/v0/hosts.py

```diff
@@ -68,13 +69,18 @@ def get(self):
         # Refer - https://www.postgresql.org/docs/13/queries-limit.html
 
         account_query = db.session.query(RhAccount.id).filter(RhAccount.account == ident['account_number']).subquery()
-        system_query = db.session.query(System.id)\
-            .filter(System.account_id.in_(account_query)).subquery()
+        if filter_display_name:
+            system_query = db.session.query(System.id)\
+                .filter(System.account_id.in_(account_query))\
+                .filter(System.display_name == filter_display_name)
```

Comment:
With this change, end user needs to pass whole `display_name` to get matching systems. Is it correct?
OR need a full-text search within this field?

Reply:
Yes, currently end user needs to pass the whole `display_name` to get the matched system.

Reply:
@apuntamb,
It would be good to have `full text + lowercase`. 

For lowercase, you can check on `ilike` - https://docs.sqlalchemy.org/en/13/orm/internals.html#sqlalchemy.orm.PropComparator.ilike

For full search, you can make use of `%` escape character. Example - "%foobar%"

Reply:
done.